docs/recurring.md

```diff
@@ -0,0 +1,183 @@
+# Recurring flow
+
+The Recurring flow enables shopper to create and save their CIT(customer initial transaction) and MIT(merchant initial transaction) payment consent which could be use to payment in the future transaction. 
+
+### Create payment consent 
+This create consent flow could be achieved by Card element or Split card element. We could either simply create a consent or create a consent while confirm an intent with this consent.
+
+### Confirm payment using consent
+For MIT subsequent transaction, merchant just need to call confirm API to finish payment.
+For CIT subsequent transaction, merchant need query and list available consents and render cvc element to let shopper choose consent and input cvc number to finish transaction. 
+
+![](assets/splitcard.gif)
+\* _An example of a split card integration. Can be customized._
+
+![](assets/card.png)
+\* _An example of a card element. Can be customized._
+
+## Guide for `Create payment consent`
+
+The following steps demonstrates the best practices to integrating with our payment platform. Code is in Javascript.
+
+Want more details? See the integration in [React](/integrations/react/src/components/SplitCard.jsx).
+
+### 1. Follow the split card element / card element integration document step 1-5
+
+You can choose the split card element or card element to setup the recurring flow. Follow the steps 1-5 in the integration doc.
+
+[Split card element integration guide](https://github.com/airwallex/airwallex-payment-demo/blob/master/docs/splitcard.md#guide)
+
+[Card element integration guide](https://github.com/airwallex/airwallex-payment-demo/blob/master/docs/card.md#guide)
+ 
+### 2. Add a button handler to trigger the recurring request and listen to the request response.
+
+This handler is called when a customer is ready to create a consent along with the necessary payload according to your requirement.
+
+```js
+// STEP #6a: Add a button handler
+document.getElementById('submit').addEventListener('click', () => {
+  Airwallex.createPaymentConsent({
```

Comment:
will this also confirm the payment intent?

Reply:
depends on whether the merchant provide the intent_id attribute